bot/cogs/nasa.py

```diff
@@ -0,0 +1,167 @@
+import os
+import random
+
+import aiohttp
+
+from discord import Color, Embed
+from discord.ext.commands import (
+    Bot,
+    BucketType,
+    Cog,
+    Context,
+    command,
+    cooldown
+)
+
+NASA_API = os.getenv("NASA_API")
+
+
+def remove_tags(text: str) -> None:
+    import re
+    clean = re.compile('<.*?>')
+    return re.sub(clean, '', text)
+
+
+class Nasa(Cog):
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+
+    @command()
+    @cooldown(16, 60, BucketType.guild)
+    async def apod(self, ctx: Context) -> None:
```

Comment:
`apod` is an unintuitive name for "astronomy picture of the day".
Something like `astronomy_picture` would be better